segments-guide.md

```diff
@@ -0,0 +1,327 @@
+
+## Templates
+
+Adobe provides many predefined segments, or templates, for clients in specific situations. Although original templates cannot be modified, copies of them can be made and then the copies can be modified. If you want to work with segments templates programmatically, you can identify them by the additional attribute `template: true` as shown at the bottom of the following example:
+
+```
+
+{
+      "id": "Has_An_Action",
+      "name": "Has an Action",
+      "description": null,
+      "definition": {
+        "container": {
+          "func": "container",
+          "pred": {
+            "val": {
+              "func": "attr",
+              "name": "variables/mobileaction"
+            },
+            "func": "exists",
+            "description": "Action Name"
+          },
+          "context": "hits"
+        },
+        "func": "segment",
+        "version": [
+          1,
+          0,
+          0
+        ]
+      },
+      "template": true
+},
+
+```
+For more information on segments templates, see [Adobe Help](https://marketing.adobe.com/resources/help/en_US/analytics/segment/seg_build.html).
+
+### Expansions
+
+Segments endpoints support the URL query parameter `expansion`. This parameter allows specifying additional data fields to be populated in response objects. Available expansions are shown in the [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/) description. The following table describes expansion fields:
+
+|           Field       | Description                                                                                                          |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| reportSuiteName           | The name of the report suite                                                        
+| ownerFullName    | The name of the owner of the segment                                                                                |
+| modified          | The date the segment was last modified                                                                   |
+| tags             |  Tags applied to the segment                                                             |
+| compatibility      | The Analytics products compatible with this segment                                                                  |
+| definition             | The [segment definition](segments.md)
+
+### Locale
+
+Segment endpoints support the URL query parameter `locale`. Supported values are `en_US`, `fr_FR`, `jp_JP`, `ja_JP`, `de_DE`, `es_ES`, `ko_KR`, `pt_BR`, `zh_CN`, and `zh_TW`. This query parameter specifies the language for responses. For example, the `en_US` localized name for the `Abandon_Cart` segment is returned as *Abandon Cart*. With the `es_ES` (Spanish) value, the same segment is returned as *Vaciar carro*.
```

Comment:
Is there somewhere that we could link to for supported locales? This type of information goes out of date pretty quickly and may be better if it could be managed in one place.

Reply:
That's a great point. However, I am having a hard time finding ANY documentation in help about locales for segments, much less the supported ones? If we don't find any, maybe I should simply delete this sentence so it won't be outdated soon?

Reply:
We don't add locales that often so this is probably fine for now. I'm just trying to make it easier to maintain by making it so that there is only one place listing all of the supported locales.